pyresample/bucket/__init__.py

```diff
@@ -0,0 +1,305 @@
+# pyresample, Resampling of remote sensing image data in python
+#
+# Copyright (C) 2019  Pytroll developers
+#
+# Author(s):
+#
+#   Panu Lahtinen <panu.lahtinen@fmi.fi>
```

Comment:
I'd like if we stopped putting individual authors in the copyright header and changed "Pytroll developers" to "Pyresample developers".

Reply:
Ok. Just took the header from some other file.